src/GitHub/Request.hs

```diff
@@ -256,8 +256,8 @@ makeHttpRequest auth r = case r of
     setAuthRequest _                                  = id
 
     getOAuthHeader :: Auth -> RequestHeaders
-    getOAuthHeader (OAuth token) = [("Authorization", BS8.pack ("token " ++ token))]
-    getOAuthHeader _                   = []
+    getOAuthHeader (OAuth token) = [("Authorization", "token " <> token)]
+    getOAuthHeader _             = []
```

Comment:
@zudov the deprecated authentication is to use **github.com** (i.e. this site) to authenticate in `github.yourcompany.com` enterprise installation.